### DISCUSSION TITLE
不推荐用这个

### DISCUSSION THREAD

**Discussion by vipsoda** [nxtrace/NTrace-core#154] [Ideas]
## 你正在使用哪个版本的 nexttrace？

linux_amd64 

## 你看到的异常现象是什么？

besttrace准度远超nexttrace

## 你期待看到的正常表现是怎样的？
至少和besttrace一样准

**Comment by tsosunchia** [comment]
这是issue区，不是complain区
请提出建设性意见，此issue -> discussion

**Comment by vipsoda** [comment]
besttrace准度都达不到 就别推了吧 根本不能用
